pkg/client/cli/cmd/helm.go

```diff
@@ -108,12 +106,6 @@ func (ha *HelmCommand) addValueSettingFlags(flags *pflag.FlagSet) {
 	}
 }
 
-func (ha *HelmCommand) addCRDsFlags(flags *pflag.FlagSet) {
-	if HelmExtendFlagsFunc != nil {
```

Comment:
At this point, we should probably just remove `HelmExtendFlagsFunc` altogether as it's not consumed anymore.